tests/scripts/create_coverag_report.py

```diff
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+import os
+from typing import Dict
+from bluechi_test.util import read_file
+
+from bluechi_test.test import BluechiTest
+from bluechi_test.container import BluechiControllerContainer, BluechiNodeContainer
+from bluechi_test.config import BluechiControllerConfig, BluechiNodeConfig
+
+
+def merge_all_info_files(start_path):
+    add_merged_file = ""
+    path_to_info_files = "data/bluechi-coverage"
+    for path in os.listdir(start_path):
+        joined_path = os.path.join(start_path, path)
+        if os.path.isdir(joined_path):
+            bluechi_coverage_location = os.path.join(joined_path, path_to_info_files)
+            if os.path.isdir(bluechi_coverage_location):
+                for path in os.listdir(bluechi_coverage_location):
+                    if "info" in path:
+                        info_file = os.path.join(bluechi_coverage_location, path)
+                        if os.path.isfile(info_file):
+                            if os.stat(info_file).st_size != 0:
+                                os.system(
+                                    f'lcov --add-tracefile {info_file} {add_merged_file} -o merged.info > /dev/null')
+                                add_merged_file = "-a merged.info"
+
+
+def exec(ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer]):
+    path_to_tests_results = os.popen(
+        "echo $TMT_TREE").read().split('/tree')[0] + "/execute/data/guest/default-0/tests/tier0/"
```

Comment:
Can't we use `os.environ` python function to get `TMT_TREE` variable value into python code?